ArduPlane/Parameters.cpp

```diff
@@ -5,11 +5,11 @@
  *
  */
 
-#define GSCALAR(v, name, def) { plane.g.v.vtype, name, Parameters::k_param_ ## v, &plane.g.v, {def_value : def} }
-#define ASCALAR(v, name, def) { plane.aparm.v.vtype, name, Parameters::k_param_ ## v, (const void *)&plane.aparm.v, {def_value : def} }
-#define GGROUP(v, name, class) { AP_PARAM_GROUP, name, Parameters::k_param_ ## v, &plane.g.v, {group_info : class::var_info} }
-#define GOBJECT(v, name, class) { AP_PARAM_GROUP, name, Parameters::k_param_ ## v, (const void *)&plane.v, {group_info : class::var_info} }
-#define GOBJECTN(v, pname, name, class) { AP_PARAM_GROUP, name, Parameters::k_param_ ## pname, (const void *)&plane.v, {group_info : class::var_info} }
+#define GSCALAR(v, name, def)           { name, &plane.g.v,                   {def_value : def},              0, Parameters::k_param_ ## v,     plane.g.v.vtype }
```

Comment:
better for these to move to AP_Param.h, and have a #define PARAM_VEHICLE_TYPE for the name of the vehicle object

Reply:
Done